src/components/Modals/Send/hooks/useFormSend/useFormSend.tsx

```diff
@@ -40,18 +41,19 @@ export const useFormSend = () => {
       try {
         const adapter = chainAdapterManager.get(data.asset.chainId) as ChainAdapter<KnownChainIds>
         if (!adapter) throw new Error(`useFormSend: no adapter available for ${data.asset.chainId}`)
+
         const value = bnOrZero(data.cryptoAmount)
           .times(bnOrZero(10).exponentiatedBy(data.asset.precision))
           .toFixed(0)
 
-        const adapterType = adapter.getChainId()
+        const chainId = adapter.getChainId()
 
         const { estimatedFees, feeType, address: to } = data
 
         const result = await (async () => {
-          if (supportedEvmChainIds.includes(adapterType)) {
+          if (supportedEvmChainIds.includes(chainId)) {
             if (!supportsETH(wallet))
-              throw new Error(`useFormSend: wallet does not support ethereum`)
+              throw new Error(`useFormSend: wallet does not have evm support`)
```

Comment:
Not sure about that change: `supportsEth` does precisely that, Ethereum feature detection. If you consider Tally Cash, it does support Mainnet, but does not `supportsEthSwitchChain()` and currently does not support any other EVM chain.

Reply:
address in: https://github.com/shapeshift/web/pull/2280/commits/8c1f4042d3807963c1fc1adbb6141f175295a97e